server/sandbox_run_linux.go

```diff
@@ -1005,6 +1020,30 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 	return resp, nil
 }
 
+// linkLogs bind mounts the pods log directory under the specified empty dir volume
+func linkLogs(kubePodUID, emptyDirVolName, namespace, kubeName, mountLabel string) error {
+	emptyDirLoggingVolumePath := podEmptyDirPath(kubePodUID, emptyDirVolName)
+	if _, err := os.Stat(emptyDirLoggingVolumePath); !os.IsNotExist(err) {
+		logDirMountPath := filepath.Join(emptyDirLoggingVolumePath, "logs")
```

Comment:
should we use SecureJoin here because we're taking user input? or should we sanitize the emptyDirVolName to ensure it's just a volume name and not anything with `../` or something?

Reply:
Added validation for empty dir that is exactly what k8s does.